Source/SwiftLintBuiltInRules/Rules/Idiomatic/NoMagicNumbersRule.swift

```diff
@@ -196,4 +202,22 @@ private extension ExprSyntaxProtocol {
         }
         return nil
     }
+
+    func isSecondOperandOfLeftShiftOperation() -> Bool {
```

Comment:
Any reason to not just do Right sift, also?

Rename method to "isSecondOperandOfShiftOperation", and look for ">>" also?

Reply:
ok - so now we'll allow **either** operand of a bitwise shift operation to be magic